fineract-provider/src/main/java/org/apache/fineract/notification/config/MessagingConfiguration.java

```diff
@@ -44,7 +44,7 @@ public class MessagingConfiguration {
     private Environment env;
 
     @Autowired
-    private NotificationEventListener notificationEventListener;
+    private ActiveMQNotificationEventListener notificationEventListener;
```

Comment:
This is fine but I'd love to get rid of the spring profile for this functionality and replace it with proper env variable configuration option.

This was rather a note and not a requirement from my side to get this PR in.

Reply:
I understood